### PR TITLE
Add command to update invalid phone numbers

### PR DESCRIPTION
**Pull request by mkangia**
https://dimagi-dev.atlassian.net/browse/ICDS-555

The script i used to update phone numbers on cases, moved that as a management command because this now needs to be run every 3 months
The only change is that now we are fetching
- person case ids
- modified in last 100 days

[Original Script](https://gist.github.com/mkangia/e4841308b7b2ff5ef03598d0f6b47f34) but was modified a bit to handle errors and then resume after some wait.

once we have instances of this being run successfully, we move this as a task that runs by itself each quarterly and notifies outcome via email

### CODE REVIEW

File: custom/icds/management/commands/sanitize_phone_numbers.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+DATE_OF_REGISTRATION_PROPERTY = "date_of_registration"
+PHONE_NUMBER_PROPERTY = "contact_phone_number"
+HAS_MOBILE_PROPERTY = "has_mobile"
+HAS_MOBILE_PROPERTY_NO_VALUE = "no"
+CASE_ITERATION_COUNT = 10000
+MAX_RESCUE_EXCEPTIONS_ON_UPDATE = 5
+CSV_HEADERS = ['Case Id']
+
+TEST_STATES = ['Test State', 'Test State 2', 'VL State', 'Trial State', 'Uttar Pradesh_GZB', 'AWW Test State']
```
Comment (biyeun) [inline custom/icds/management/commands/sanitize_phone_numbers.py]: nitpick...i think the convention is to format lists like this for readability
```
TEST_STATES = [
    'Test State', 
    'Test State 2', 
    'VL State', 
    'Trial State', 
    'Uttar Pradesh_GZB', 
    'AWW Test State',
]
```

File: custom/icds/management/commands/sanitize_phone_numbers.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            start_date=start_date
+        )
+
+        filename = 'invalid_phone_numbers_with_91_part_%s_%s.csv' % (self.db_alias, datetime.utcnow())
+        with open(filename, 'w+b') as output:
+            cases_iterated = 0
+            writer = csv.writer(output)
+            writer.writerow(CSV_HEADERS)
+            if self.log_progress:
+                print('iterating now')
```
[the remark this answers is not in the document]
Comment (biyeun) [reply custom/icds/management/commands/sanitize_phone_numbers.py]: instead of `print` use `self.stdout.write('iterating now\n')`

**Review by biyeun — APPROVED** [review]
some non blocking comments that would be a nice to address

**Comment by mkangia** [comment]
Thanks @biyeun, updated both

**Pull request merged**